packages/examples/cvat/recording-oracle/src/modules/webhook/jobs/process_exchange_oracle_webhooks.py

```diff
@@ -64,14 +62,15 @@ def process_exchange_oracle_webhooks() -> None:
                     escrow.store_results(
                         webhook.chain_id,
                         webhook.escrow_address,
-                        f"{StorageConfig.bucket_url()}{files[0]}",
+                        f"{StorageConfig.bucket_url()}{files[0]['key']}",
                         files[0],
                     )
 
                     db_service.create_webhook(
                         session,
                         webhook.escrow_address,
                         webhook.chain_id,
+                        webhook.s3_url,
```

Comment:
Please, change it to the url to recording oracle results.  
`webhook.s3_url` is the url to exchange oracle results.